pyfolio/utils.py

```diff
@@ -117,8 +118,9 @@ def get_symbol_rets(symbol):
             if datetime.now() - pd.to_datetime(
                     getmtime(filepath), unit='s') < pd.Timedelta(days=1):
                 rets = pd.read_hdf(filepath, 'df')
-        except:
-            pass
+        except Exception as e:
```

Comment:
Sure. I'll also add some way to force-download the benchmark in case of exception.